manifests/init.pp

```diff
@@ -195,6 +199,7 @@
   Boolean $manage_git                                         = false,
   Boolean $manage_virtualenv                                  = false,
   String[1]  $virtualenv_version                              = $puppetboard::params::virtualenv_version,
+  Optional[String] $virtualenv_dir                            = $puppetboard::params::virtualenv_dir,
```

Comment:
instead of setting the static value in params.pp, please directly add it here. When we in the future switch to puppet-strings, puppet-strings will be able to read the value from init.pp. Also I think we can switch the datatype to Stdlib::Absolutepath?
```suggestion
  Stdlib::Absolutepath $virtualenv_dir                        = $puppetboard::params::virtualenv_dir,
```